### DISCUSSION TITLE
Custom Tooltips

### DISCUSSION THREAD

**Discussion by paur94** [recharts/recharts#4028] [Q&A]
Hey everyone,
I've been working with the stacked bar chart and ran into a question regarding stacked bar charts and tooltips. Specifically, I'm wondering if it's possible to display different tooltips when hovering over different stacks within a stick.
Thanks.

**Comment by ckifer** [comment]
You might be able to make custom tooltips work for this with conditional content

If you do similar to this example but check the payload for which stack you are hovering you may be able to get close
https://github.com/recharts/recharts/blob/master/storybook/stories/Examples/Tooltip.stories.tsx#L178-L240
